ClientGUI/XNAClientCheckBox.cs

```diff
@@ -13,19 +13,26 @@ public XNAClientCheckBox(WindowManager windowManager) : base(windowManager)
         {
         }
 
+        private void CreateToolTip()
+        {
+            if (ToolTip == null)
+                ToolTip = new ToolTip(WindowManager, this);
+        }
```

Comment:
IMO at this point we should move `ToolTip` even more higher in the hierarchy, so it can be defined optionally on every element.

Reply:
I agree, but it's best to leave it to another branch and time.